src/main/java/uk/gov/justice/laa/amend/claim/controllers/HomePageController.java

```diff
@@ -10,4 +10,9 @@ public class HomePageController {
     public String onPageLoad() {
         return "index";
     }
+
+    @GetMapping("/throw-error")
+    public String sentryTest() {
+        throw new RuntimeException("This is a test exception");
+    }
```

Comment:
Do we want to be exposing this endpoint? Perhaps we could make this a local thing only?